.changeset/large-maps-shout.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Could you move this into the other changeset?

Reply:
It should just be removed
We don't need to update the version or changelog of the test app

Reply:
Sure! removed.